pkg/networkserver/grpc_deviceregistry.go

```diff
@@ -1426,27 +1507,12 @@ func (ns *NetworkServer) Set(ctx context.Context, req *ttnpb.SetEndDeviceRequest
 	if st.HasSetField(
 		"frequency_plan_id",
 		"lorawan_phy_version",
-		"mac_settings.adr.mode.disabled",
```

Comment:
These parameters do not depend on band specific settings and were wrongly specified here. Hence, they were removed.